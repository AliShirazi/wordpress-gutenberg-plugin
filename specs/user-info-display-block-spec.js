import {
  enablePageDialogAccept,
  createNewPost,
  getEditedPostContent,
  insertBlock
} from '@wordpress/e2e-test-utils'

describe('User Gutenberg Block', () => {
  beforeAll(async () => {
    await enablePageDialogAccept()
  })
  beforeEach(async () => {
    await createNewPost()
  })

  it('User Info Display Block successfully added to page', async () => {
    await insertBlock('User Info Display')
    // Check if block was inserted
    expect(await page.$('[data-type="wp-engine/gutenberg"]')).not.toBeNull()
    expect(await getEditedPostContent()).toMatchSnapshot()
  })
})
