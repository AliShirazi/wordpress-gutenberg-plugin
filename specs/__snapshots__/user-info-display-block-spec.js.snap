// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User Gutenberg Block User Info Display Block successfully added to page 1`] = `
"<!-- wp:wp-engine/gutenberg {\\"userGravatarUrl\\":\\"http://www.gravatar.com/avatar/?d=mp\\",\\"userBio\\":\\"Select A User\\",\\"userName\\":\\"Select A User\\"} -->
<div class=\\"wp-block-wp-engine-gutenberg\\" style=\\"color:#000000;background-color:#FFFFFF\\"><div class=\\"block-section\\"><img src=\\"http://www.gravatar.com/avatar/?d=mp\\"/></div><div class=\\"block-section\\"><p>Select A User</p></div><div class=\\"block-section\\"><p>Select A User</p></div></div>
<!-- /wp:wp-engine/gutenberg -->"
`;
